src/drawers/areaDrawer.ts

```diff
@@ -11,9 +11,15 @@ export module Drawers {
       this._areaSelection.datum(data);
     }
 
-    public setup(area: d3.Selection<void>) {
-      Drawer.prototype.setup.call(this, area);
+    public renderArea(): d3.Selection<void>;
+    public renderArea(area: d3.Selection<void>): Drawer;
+    public renderArea(area?: d3.Selection<void>): any {
+      if (area == null) {
+        return super.renderArea();
+      }
+      Drawer.prototype.renderArea.call(this, area);
```

Comment:
Oh... Well, in that case `Drawers.Area` should not inherit from `Drawers.Line`. I see no point in keeping consistency with plot inheritance